tests/libraries/configuration.py

```diff
@@ -75,12 +82,18 @@ def create(self):
 
         """
         # Initialize key variables
-        config_file = '{}{}pattoo.yaml'.format(self._config_directory, os.sep)
+        base_config = '{}{}pattoo.yaml'.format(self._config_directory, os.sep)
+
+        agent_config = '{}{}pattoo_agent.yaml'.format(
+                                            self._config_directory, os.sep)
 
-        # Write good_config to file
-        with open(config_file, 'w') as f_handle:
+        # Write to config files to file
+        with open(base_config, 'w') as f_handle:
             yaml.dump(self._config, f_handle, default_flow_style=False)
 
+        with open(agent_config, 'w') as f_handle:
```

Comment:
I discovered this preferred way of accessing files with error handling: https://docs.python.org/3/library/os.html [Search for os.access](https://docs.python.org/3/library/os.html)


```
    try:
        fh_ = open(filename, 'rb')
    except PermissionError:
        # Do something
        log.log2(....
    except FileNotFoundError:
        # Do something
        log.log2(....
    except:
        # Do something
        log.log2(....
    else:
        with fh_:
            yaml.dump(self._config, fh_, default_flow_style=False)

```